doc/source/georeferencing.md

```diff
@@ -0,0 +1,214 @@
+---
+file_format: mystnb
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: geoutils-env
+  language: python
+  name: geoutils
+---
+(georeferencing)=
+# Referencing
+
+Below, a summary of the **georeferencing attributes** of geospatial data objects and the **methods to manipulate these 
+georeferencing attributes** in different projections, without any data transformation. For georeferenced transformations 
+(such as reprojection, cropping), see {ref}`geotransformations`.
+
+```{code-cell} ipython3
+:tags: [remove-cell]
+
+# To get a good resolution for displayed figures
+from matplotlib import pyplot
+pyplot.rcParams['figure.dpi'] = 600
+pyplot.rcParams['savefig.dpi'] = 600
+pyplot.rcParams['font.size'] = 9
+```
+
+## Attributes
+
+In GeoUtils, the **georeferencing syntax is consistent across all geospatial data objects**. Additionally, **data objects 
+load only their metadata by default**, allowing quick operations on georeferencing without requiring the array data 
+(for a {class}`~geoutils.Raster`) or geometry data (for a {class}`~geoutils.Vector`) to be present in memory.
+
+### Metadata summary
+
+To summarize all the metadata of a geospatial data object, including its georeferencing, {func}`~geoutils.Raster.info` can be used:
+
+
+```{code-cell} ipython3
+:tags: [hide-cell]
+:mystnb:
+:  code_prompt_show: "Show the code for opening example files"
+:  code_prompt_hide: "Hide the code for opening example files"
+
+import geoutils as gu
+rast = gu.Raster(gu.examples.get_path("exploradores_aster_dem"))
+vect = gu.Vector(gu.examples.get_path("exploradores_rgi_outlines"))
+```
+
+```{code-cell} ipython3
+# Print raster and vector info
+rast.info()
+vect.info()
+```
+
+### Coordinate reference systems
+
+[Coordinate reference systems (CRSs)](https://en.wikipedia.org/wiki/Spatial_reference_system), sometimes also called a 
```

Comment:
I woudl just write CRS instead of CRSs (even though it's plural here)

Reply:
Checked grammar recommendations (it's not very intutive): "Acronyms are made plural by adding an “s” at the end if there are no periods in the acronym", so leaving like this!